.github/release.config.js

```diff
@@ -15,10 +15,10 @@ module.exports = {
             channel: "zowe-v1-lts",
             level: "patch",
         },
-        // {
-        //     name: "next",
-        //     prerelease: true,
-        // }
+        {
+            name: "next",
+            prerelease: true,
```

Comment:
No need to change this, but wondering if we should be explicit with our `channel`s?

Reply:
Sorry for the delay 😢 The `channel` defaults to the `name`, but its probably good practice to be explicit like we are for the other branches 😋 We could add `channel: "next"` like the following:
```javascript
    {
        name: "next",
        channel: "next",
        prerelease: true
    }
```